src/tests/Microsoft.PowerFx.Core.Tests.Shared/ExpressionTestCases/AsType_UO.txt

```diff
@@ -0,0 +1,94 @@
+#SETUP: AllowTypeLiteral
+
+// Primitives
+>> AsType(ParseJSON("987654321"), Number)
+987654321
+
+>> AsType(ParseJSON("98765.4321"), Number) > AsType(ParseJSON("98765.4320"), Number)
+true
+
+>> AsType(ParseJSON("""AstypeFunction"""), Type(Text))
+"AstypeFunction"
+
+>> AsType(ParseJSON("""1984-01-01"""), Date)
+Date(1984,1,1)
+
+>> AsType(ParseJSON("""1900-12-31T23:59:59.999Z"""), DateTime)
+DateTime(1900,12,31,23,59,59,999)
```

Comment:
Can we flip the types on those two? For example:

    >> AsType(ParseJSON("""1984-01-01"""), DateTime)
    Date(1984,1,1,0,0,0)

    >> AsType(ParseJSON("""1900-12-31T23:59:59.999Z"""), Date)
    DateTime(1900,12,31)